### ISSUE TITLE
Flag --incompatible_config_setting_private_default_visibility will break rules_webtesting in Bazel 7.0

### ISSUE THREAD

**Issue by keertk** [bazelbuild/rules_webtesting#446]
Incompatible flag `--incompatible_config_setting_private_default_visibility` will be enabled by default in the next major release (Bazel 7.0), thus breaking rules_webtesting. Please migrate to fix this and unblock the flip of this flag.

The flag is documented here: https://github.com/bazelbuild/bazel/issues/12933.

Please check the following CI builds for build and test results:

- [MacOS, OpenJDK 11](https://buildkite.com/bazel/bazelisk-plus-incompatible-flags/builds/1341#0184a208-6e92-40c3-b3a7-0f44d7b83533)
- [Ubuntu 18.04, OpenJDK 11](https://buildkite.com/bazel/bazelisk-plus-incompatible-flags/builds/1341#0184a208-6e8f-44d0-9a68-e8b7c7409ba5)
- [Windows, OpenJDK 11](https://buildkite.com/bazel/bazelisk-plus-incompatible-flags/builds/1341#0184a208-6e95-41e2-967b-303db69ae1ac)

Never heard of incompatible flags before? We have [documentation](https://docs.bazel.build/versions/master/backward-compatibility.html) that explains everything.
If you have any questions, please file an issue in https://github.com/bazelbuild/continuous-integration.

**Comment by joshbruning** [comment]
This project is on an old rules_go release; after its deps are updated, I think this will be ok

**Comment by joshbruning** [comment]
I tried building with today's rules_go release; however, there will need to also be a new bazel-gazelle (not yet released) due to (at least) https://github.com/bazelbuild/rules_go/pull/3276

**Comment by alexeagle** [comment]
@joshbruning hi, are you still a maintainer here? I noticed this issue is 6mo old, I am sure any changes you need from rules_go and bazel-gazelle have been released.

**Comment by joshbruning** [comment]
No, I haven't been for some years; @mtrea - I think you should try updating the version set and trying a build at least (it may end up being easy); the version diff is getting bigger, and will eventually make the update more difficult to do in one-shot, and there have been an increasing number of linter headaches

**Comment by sgowroji** [comment]
Hi @joshbruning, Could you please look in to this issue.Thanks!

**Comment by joshbruning** [comment]
I'm not a maintainer.

**Comment by sgowroji** [comment]
@mtrea Can you Please have a look on the above failures. Thanks!
